share/ansible_navigator/ee-details/collections/ansible_collections/ansible/ansible_navigator/README.md

```diff
@@ -0,0 +1,7 @@
+# Ansible Collection - ansible.ansible_navigator
+
+http://github.com/ansible/ansible-navigator
+
+### Developer note
+
+Changes to this collection will likely result in the fixtures needing to be recreated nd running with `tox --recreate`, since `tox` does not track files outside the source tree
```

Comment:
```suggestion
Changes to this collection will likely result in the fixtures needing to be recreated and running with `tox --recreate`, since `tox` does not track files outside the source tree
```

Reply:
DOH, TY